index.js

```diff
@@ -49,16 +49,17 @@ class ReportPortalHelper extends Helper {
 
     return rpClient.startLaunch({
       name: this.config.launchName || suiteTitle,
-      start_time: rpClient.helpers.now(),
```

Comment:
@AmsterGet don't we need the `startTime` anymore?

Reply:
As you can see [here](https://github.com/reportportal/client-javascript/blob/master/lib/report-portal-client.js#L144) client add `startTime` automatically.

Reply:
cool! Thanks